src/nvim/po/CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ if(HAVE_WORKING_LIBINTL AND GETTEXT_FOUND AND XGETTEXT_PRG AND ICONV_PRG)
         -DXGETTEXT_PRG=${XGETTEXT_PRG}
         -DPOT_FILE=${NVIM_POT}
         -DSEARCH_DIR=${CMAKE_CURRENT_SOURCE_DIR}
-        "'-DSOURCES=${NEOVIM_RELATIVE_SOURCES}'"
+        "\"-DSOURCES=${NEOVIM_RELATIVE_SOURCES}\""
```

Comment:
Actually, I think you can get rid of the escaped quotes all together.  CMake's expansion rules are different than the shell, and IIRC, the whole thing will get passed as a single argument.